www/js/app.js

```diff
@@ -991,23 +1002,27 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     /**
      * Display the list of articles with the given array of DirEntry
      * @param {Array} dirEntryArray The array of dirEntries returned from the binary search
-     * @param {Object} reportingSearchPrefix The prefix of the reporting search
+     * @param {Object} reportingSearch The reporting search object
      */
-    function populateListOfArticles(dirEntryArray, reportingSearchPrefix) {
-        // Do not allow cancelled or changed searches to report
-        if (globalstate.search.status === 'cancelled' || globalstate.search.prefix !== reportingSearchPrefix) return;
-        var stillSearching = globalstate.search.status === 'interim';
+    function populateListOfArticles(dirEntryArray, reportingSearch) {
+        // Do not allow cancelled searches to report
+        if (!/cancelled/.test(reportingSearch.status)) console.log("Reporting search is: " + reportingSearch.prefix + " [" + reportingSearch.status + "]");
```

Comment:
Why are you using a regular expression here, instead of comparing the string? If you prefer to keep it, it would be safer (and faster) to match the whole string instead of a subtring : add `^` at the beginning and `$` at the end)

Reply:
Because I love regexes ;-) ... However, I think I can just remove this, as it's a diagnostic `console.log` we don't want in the final code.